resources.rst

```diff
@@ -0,0 +1,37 @@
+=========
+Resources
+=========
+
+This document lists some of the resources used to understand the theory
+behind this work and other useful links.
+
+Tuning/Intervals
+================
+
+* Why There are Twelve Notes in Music: https://www.youtube.com/watch?v=IT9CPoe5LnM
+
+  * an `octave <https://en.wikipedia.org/wiki/Octave>`_
+    is an interval with a frequency ratio of 2:1 (e.g. A4 and A5)
+  * a `perfect fifth <https://en.wikipedia.org/wiki/Perfect_fifth>`_
+    is an interval with a frequency ratio of 3:2
+  * the `Pythagorean tuning <https://en.wikipedia.org/wiki/Pythagorean_tuning>`_
+    uses the 3:2 ratio
+  * the `Equal temperament <https://en.wikipedia.org/wiki/Equal_temperament>`_
+    divides the octave in 12 equal steps
+  * if we build a scale by using a 3:2 ratio, the frequencies for the octaves
+    don't match
+  * the equal temperament is commonly used in classical and western music
```

Comment:
Two important bits that I feel are missed by this:
1) It is commonly used in Western music (with a capital W) nowadays, but it was less historically. Also, 12 tones ET wasn't the only attempt at a more regular 12 tones scale
2) Not sure what "classical and Western" means. By "classical music" we usually identify a [specific part of music produced by Western culture](https://en.wikipedia.org/wiki/Classical_music).

Reply:
I took it from the wiki page:

> In classical music and Western music in general, the most common tuning system since the 18th century has been twelve-tone equal temperament (also known as 12 equal temperament, 12-TET or 12-ET; informally abbreviated to twelve equal), which divides the octave into 12 parts, all of which are equal on a logarithmic scale, with a ratio equal to the 12th root of 2 (12√2 ≈ 1.05946).

Reply:
I removed classical and left only "Western" and "now", to clarify that this is the system that is now commonly used in Western music.  I.e. if the goal is to produce Western-sounding music, then this is the tuning to use, otherwise there are also other tunings that work differently.